README.md

```diff
@@ -430,6 +430,34 @@ This will then allow you to connect to the database, eg:
 - Change `staging` to `production` in the above commands to access production.
 - Port 5433 is used in the above examples instead of the usual 5432 for postgres, as 5432 will not work if postgres is running locally.
 
+## Anonymised database export and restore
+
+In order to create an anonymised dump of an environments database you can:
+
+```bash
+$ ./scripts/db_export.sh [environment]
+```
+Where environment is `production`, `staging` or a branch name from uat, e.g. `ap-1234`
+
+It requires that you have kubectl authenticated and your context set to the current live context
+
+It will connect to the required kubernetes namespace and pod and run the `rake db:export` task this will generate a filename of `[environment].anon.sql`
+
+It will then copy the compressed restore file to the `tmp` folder in the project and de-compress it
+
+If a file from the same environment exists, it will prompt you to overwrite the local copy
+
+It will then an output a restore command to enable you to restore it to your local psql instance at your convenience
```

Comment:
>It will then an output 
It doesn't need that 'an'

Reply:
My brain farted on adding the word `create` 🤦 
>It will then create an output...